src/artifacts/entities/artifact.entity.ts

```diff
@@ -54,12 +53,13 @@ export abstract class Artifact extends PrincipalScopedEntity {
   @Property()
   description?: string;
 
-  constructor({ thread, message, name, description, ...rest }: ArtifactInput) {
+  constructor({ thread, message, name, description, accessSecret, ...rest }: ArtifactInput) {
     super(rest);
     this.thread = thread;
     this.message = message;
     this.name = name;
     this.description = description;
+    this.accessSecret = accessSecret;
```

Comment:
@kapetr public artifacts cannot be created until this gets in.

Reply:
You need to use an update to the artifact.

Reply:
So I shall remove `public` flag from the `createArtifact` service?

Reply:
OK, we can keep the public flag in create DTO.